OpenRA.Mods.Common/Scripting/Properties/GeneralProperties.cs

```diff
@@ -72,6 +73,13 @@ public bool HasProperty(string name)
 		{
 			return Self.HasScriptProperty(name);
 		}
+
+		[Desc("Render a target flash on the actor. If set, 'asPlayer'",
+			"defines which player palette to use. Duration is in ticks.")]
+		public void FlashTarget(int duration = 4, Player asPlayer = null)
```

Comment:
This doesn't accept a target argument\ to flash, so "FlashTarget" isn't a great name.  How about just "Flash"?